test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -587,7 +587,7 @@ describe('Custom network', function () {
           fixtures: new FixtureBuilder()
             .withNetworkController({
               networkConfigurations: {
-                networkConfigurationId: {
+                networkConfigurationId2: {
```

Comment:
Curious: why was this needed?

Reply:
Without this, the test fails because arbitrum becomes the current network which cannot be deleted. I can give the ID a prettier name.